crates/aiken/src/cmd/check.rs

```diff
@@ -16,7 +16,7 @@ pub struct Args {
 
     /// Only run tests if their path + name match the given string
```

Comment:
Let's extend the documentation here to show how to use it. So that we have all information needed when running `--help`.

Reply:
Cool, good call